docs/roman_datamodels/datamodels/using_datamodels.rst

```diff
@@ -19,49 +19,54 @@ page::
     >>> dm = rdm.open('my_wonderful_roman_data.asdf')
 
     >>> type(dm)
-    roman_datamodels.datamodels.ScienceRawModel
+    <class 'roman_datamodels.datamodels._datamodels.ScienceRawModel'>
+
+    # Can view the associated schema
+    >>> dm.tag
+    'asdf://stsci.edu/datamodels/roman/tags/wfi_science_raw-1.0.0'
```

Comment:
```suggestion
    >>> dm.schema_uri
    'asdf://stsci.edu/datamodels/roman/schemas/wfi_science_raw-1.0.0'
```
Or alternatively change "schema" on line 24 to "tag".

Reply:
Changed.